src/pages/build.html

```diff
@@ -257,6 +265,37 @@ <h6>Stack:</h6>
             <ul>
               <li>Solidity</li>
             </ul>
+
+            <hr class="hr-dash my-7">
+
+            <!-- Lucidity -->
+            <h2 id="lucidity"><a href="https://lucidity.tech/" target="_blank">Lucidity</a></h2>
+            <p>
+              Lucidity is using Plasma Cash coupled with a stable coin for payments in AdTech supply chains.
+            </p>
+            <br>
+      
+            <h5 id="lucidity-plasma-mvp"><a href="https://github.com/luciditytech/lucidity-plasma" target="_blank">Plasma MVP</a></h5>
+            <p>
+              Lucidity maintains an implementation of Plasma MVP root chain contracts.
+              Particularly implements UTXO model for mass-exists.
+            </p>
+            <h6>Stack:</h6>
+            <ul>
+              <li>Solidity</li>
+              <li>Javascript</li>
+            </ul>
+
+            <h5 id="lucidity-plasma-mvp"><a href="https://github.com/luciditytech/lucidity-plasma-cash" target="_blank">Plasma Cash</a></h5>
```

Comment:
Need to change this header id from `lucidity-plasma-mvp` to `lucidity-plasma-cash`